test/test-pages/wikia/expected.html

```diff
@@ -0,0 +1,28 @@
+<div id="readability-page-1" class="page">
+    <div class="entry-content">
+        <p>Although Lucasfilm is already planning a birthday bash for the Star Wars Saga at <a href="http://starwars.wikia.com/wiki/Celebration_Orlando" data-is="trackable" riot-tag="trackable">Celebration Orlando</a> this April, fans might get another present for the saga’s 40th anniversary. According to fan site <a href="http://makingstarwars.net/2017/02/rumor-unaltered-original-star-wars-trilogy-re-released-year/" data-is="trackable" riot-tag="trackable">MakingStarWars.net</a>, rumors abound that Lucasfilm might re-release the unaltered cuts of the saga’s original trilogy.</p>
```

Comment:
Hm, we should add an issue if we haven't already to remove random unused attributes...